integration/integration_test.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2019-2021 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"testing"
+)
+
+var integration = flag.Bool("integration", false, "run integration tests")
+
+// image such as docker.io/kinvolk/gadget:latest
+var image = flag.String("image", "", "gadget container image")
+
+func TestMain(m *testing.M) {
```

Comment:
Yes, `TestMain` is run once for all the tests.
I use it to setup `inspektor-gadget` and to avoid doing it in all the tests, so shortening tests duration and reducing resource usage.